added css

kojhcd opened the pull request:
added css to div 9

NAlexPear commented on index.html:
```diff
@@ -60,68 +60,16 @@ <h1>This be Adam's box.</h1>
+              <p> Email: Kyleojennings@gmail.com</p>
+
         </div>
+        <div>TEN</div>
```
Make sure you aren't adding the `NINE`, `TEN`, and `ELEVEN` boxes back into the codebase!

kojhcd commented:
Added new background image and updated CSS.  For the life of me I could not get the css to work on my local host.  If you could take a look and see what my error may have been, that'd be great 1= D
